web/client/api/catalog/tileProvider.js

```diff
@@ -0,0 +1,90 @@
+/*
+ * Copyright 2020, GeoSolutions Sas.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+import CONFIG_PROVIDER from '../../utils/ConfigProvider';
```

Comment:
Some services are not working anymore or require an API key. I updated the ConfigProvider but I it was not enough, and I though that if these services change again, we need a way to disable them without needing of recompile.
For this reason I design a white list (configurable) of services that work. I placed there the OpenStreetMap, OpenSeaMap and Stamen as samples (1st because of large usage, 2nd because it is a sample of useful not-backgound tms, 3rd as a sample of cool new backgrounds) we could decide to add / remove services to this defaults list, balancing the risk of these services to disappear during the time.

Reply:
Ok, found